src/components/inputs/z-input/index.tsx

```diff
@@ -377,6 +377,7 @@ export class ZInput {
 
   private renderTextarea(): HTMLDivElement {
     const attributes = this.getTextAttributes();
+    const ariaLabel = this.ariaLabel ? {"aria-label": this.ariaLabel} : {};
```

Comment:
A sto punto sposterei anche l'aria-label nel metodo `getTextAttributes()` visto che la vai a settare in entrambi i punti in cui viene richiamato

Reply:
l'idea era quella, ma gli `aria-*` non vengono riconosciuti come attributi validi per `InputHTMLAttributes<HTMLInputElement | HTMLTextAreaElement>`

Reply:
Azz, ok!